### PR TITLE
feat: #188 add tests wayCard, userCars, VerticalContainer

### PR DESCRIPTION
**Pull request by AKiroi**
- created tests to userCard
- created tests to wayCard
- created tests to Vertical container

### CODE REVIEW

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/userCard/UserCard.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+  it("should render userCard correctly", () => {
+    cy.get(getDataCy(USER_CARD_CY)).should("exist");
+  });
+
+  it("should display the correct username", () => {
+    cy.get(getDataCy(USER_CARD_CY)).contains(USER_PREVIEW_DATA.name);
+  });
+
+  it("should display the correct user name", () => {
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/userCard/UserCard.cy.tsx:57]: User name or user email?)

File: mw-webapp/src/component/userCard/UserCard.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      `${USER_PREVIEW_DATA.favoriteWays.length} favorite ways`,
+    );
+  });
+
+  it("should display the correct number of mentorings", () => {
+    cy.get(getDataCy(USER_CARD_CY)).contains(
+      `${USER_PREVIEW_DATA.mentoringWays.length} mentoring ways`,
+    );
+  });
+
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/userCard/UserCard.cy.tsx:85]: You can check it in one test

File: mw-webapp/src/component/wayCard/WayCard.cy.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+  it("should display the correct creation date", () => {
+    const creationDate = DateUtils.getShortISODateValue(
+      WAY_PREVIEW_DATA.createdAt,
+    );
+    cy.get(getDataCy(WAY_CARD_CY)).contains(`Created: ${creationDate}`);
+  });
+
+  it("should display the correct owner name", () => {
+    cy.get(getDataCy(WAY_CARD_CY)).contains(WAY_PREVIEW_DATA.owner.name);
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/wayCard/WayCard.cy.tsx:127]: maybe better to check it in one test? Don't know? maybe need ask Victor or advisor QA)

File: mw-webapp/src/component/userCard/UserCard.cy.tsx (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/component/userCard/UserCard.cy.tsx]: fixed

File: mw-webapp/src/component/wayCard/WayCard.cy.tsx (hunk 3 above)
Comment (AKiroi) [reply mw-webapp/src/component/wayCard/WayCard.cy.tsx]: fixed. I combined everything into one test "should display the correct content elements"

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**